lib/puppet/provider/service/upstart.rb

```diff
@@ -1,5 +1,3 @@
-require 'semver'
```

Comment:
Not sure how to find the answer to that question. Is there any way to efficiently search the code of all forge modules? I've been wanting to do such searches many times before.